setup-ovs.sh

```diff
@@ -45,8 +45,7 @@ if [[ $(nmcli conn | grep -c ovs) -eq 0 ]]; then
                  conn.interface brcnv master brcnv-port \
                  ipv4.method auto \
                  ipv4.dhcp-client-id "mac" \
-                 connection.autoconnect no \
-                 802-3-ethernet.cloned-mac-address $mac
+                 connection.autoconnect no
```

Comment:
I'm confused, don't we want to remove this script in this branch?

Reply:
Yes we did :) 
But I don't have permissions to delete files in the repo 

<img width="1043" alt="Screen Shot 2021-04-27 at 16 53 25" src="https://user-images.githubusercontent.com/42510468/116253396-1e875880-a779-11eb-8ee9-b3708170dfe8.png">

Reply:
Sure you do - push a commit removing them

Reply:
I pushed the comment with git rm to the file and I get this message that I don't have permissions

Reply:
Could you just delete those files and do `git add`?

Reply:
This is what I did, have a look at this: 
https://github.com/kobi86/rhcos-slb/tree/linux-bridge

Reply:
Just to be clear, you did not `git rm` there?

Reply:
What? this is the repo from where I am creating the PR...

Reply:
Could you try rebasing your repo/branch on the current HEAD of this repo? I think that may be all that is missing.

The note about "write access" is only related to the fact that you can't click the merge button AFAIK.